src/main/resources/schema.sql

```diff
@@ -0,0 +1,47 @@
+
+CREATE DATABASE IF NOT EXISTS airlines;
+CREATE USER 'user'@'localhost' IDENTIFIED BY 'user';
+GRANT ALL PRIVILEGES ON airlines.* TO 'user'@'localhost';CREATE DATABASE IF NOT EXISTS airlines;
```

Comment:
this part `CREATE DATABASE IF NOT EXISTS airlines;` should be on separate line

Reply:
fixed in